package.json

```diff
@@ -23,7 +23,7 @@
     "test.spec": "stencil test --spec"
   },
```

Comment:
Do we want source map generation? This is enabled by default in Stencil 3. I recall there being some issues in Framework that made us disable it.

Reply:
Good call, I disabled it for the same reasoning we disabled it for PWA elements: https://github.com/ionic-team/pwa-elements/pull/125